pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -154,6 +154,59 @@
     dependencies = with super; [ plenary-nvim ];
   };
 
+  avante-nvim = let
+    version = "0.0.2";
+    src = fetchFromGitHub {
+      owner = "yetone";
+      repo = "avante.nvim";
+      rev = "refs/tags/v${version}";
+      hash = "sha256-p0ILVK1nADMFQ2FCVgWKouoEr+Fenf62p+RMgeRCx7E=";
+    };
+    avante-libs = rustPlatform.buildRustPackage {
+      pname = "avante-libs";
+      inherit version src;
+
+      nativeBuildInputs = [ pkg-config ];
+
+      buildInputs = [ openssl ];
+
+      buildFeatures = let
+        luaVersion = luaPackages.lua.version;
+        major = lib.versions.major luaVersion;
+        minor = lib.versions.minor luaVersion;
+      in [ "lua${major}${minor}" ];
+
+      cargoLock = {
+        lockFile = ./avante-nvim/Cargo.lock;
+        outputHashes = {
+          "mlua-0.10.0-beta.1" = "sha256-ZEZFATVldwj0pmlmi0s5VT0eABA15qKhgjmganrhGBY=";
+        };
+      };
+    };
+  in
+    buildVimPlugin {
+      pname = "avante.nvim";
+      inherit version src;
+
+      dependencies = with super; [ dressing-nvim nui-nvim plenary-nvim ];
+
+      postInstall = let
+        extension = if stdenv.isDarwin then "dylib" else "so";
+      in ''
+        mkdir $out/build
+        cp ${avante-libs}/lib/libavante_templates.${extension} $out/build/avante_templates.${extension}
+        cp ${avante-libs}/lib/libavante_tokenizers.${extension} $out/build/avante_tokenizers.${extension}
+      '';
+
+      doInstallCheck = true;
+      nvimRequireCheck = "avante";
+
+      meta = {
+        homepage = "https://github.com/yetone/avante.nvim/";
+        changelog = "https://github.com/yetone/avante.nvim/releases/tag/v${version}";
```

Comment:
Would be great to add `meta.maintainers`

Reply:
Done